src/test/java/org/openrewrite/java/testing/junit6/MinimumJreConditionsTest.java

```diff
@@ -0,0 +1,843 @@
+          "JRE.JAVA_8",
+          "value = JRE.JAVA_11",
+          "value = JRE.JAVA_8",
+          "versions = { 11 }",
+          "versions = { 8 }"
+        })
+        void removeTestEnabledOnOlderJre(String jre) {
+            rewriteRun(
+              java(
+                """
+                  import org.junit.jupiter.api.Test;
+                  import org.junit.jupiter.api.condition.EnabledOnJre;
+                  import org.junit.jupiter.api.condition.JRE;
+
+                  import static org.junit.jupiter.api.condition.JRE.*;
+
+                  class MyTest {
+                      @Test
+                      @EnabledOnJre(%s)
+                      void testOnlyOnJava8() {
+                          System.out.println("Java 8");
+                      }
+
+                      @Test
+                      void normalTest() {
+                          System.out.println("Any version");
+                      }
+                  }
+                  """.formatted(jre),
+                """
+                  import org.junit.jupiter.api.Test;
+                  import org.junit.jupiter.api.condition.EnabledOnJre;
+                  import org.junit.jupiter.api.condition.JRE;
+
+                  import static org.junit.jupiter.api.condition.JRE.*;
+
+                  class MyTest {
+
+                      @Test
+                      void normalTest() {
+                          System.out.println("Any version");
+                      }
+                  }
+                  """
+              )
+            );
+        }
+
+        @ParameterizedTest
+        @ValueSource(strings = {
+          "JAVA_17",
+          "JRE.JAVA_17",
+          "value = JRE.JAVA_17",
+          "versions = 17",
+          "versions = { 17 }"
+        })
+        void removeAnnotationEnabledOnJava17(String jre) {
+            rewriteRun(
+              java(
+                """
+                  import org.junit.jupiter.api.Test;
+                  import org.junit.jupiter.api.condition.EnabledOnJre;
+                  import org.junit.jupiter.api.condition.JRE;
+
+                  import static org.junit.jupiter.api.condition.JRE.*;
+
+                  class MyTest {
+                      @Test
+                      @EnabledOnJre(%s)
+                      void testOnJava17() {
+                          System.out.println("Java 17");
+                      }
+                  }
+                  """.formatted(jre),
+                """
+                  import org.junit.jupiter.api.Test;
+                  import org.junit.jupiter.api.condition.EnabledOnJre;
+                  import org.junit.jupiter.api.condition.JRE;
+
+                  import static org.junit.jupiter.api.condition.JRE.*;
+
+                  class MyTest {
+                      @Test
+                      void testOnJava17() {
+                          System.out.println("Java 17");
+                      }
+                  }
+                  """
```

Comment:
Seeing this now we should probably retain these, as the tests might just as well run on Java 21 or 25; Java 17 is the new minimum version supported, but not guaranteed to be the version used.
```suggestion
                  """.formatted(jre)
```